scripts/jenkins/Jenkinsfile

```diff
@@ -902,6 +923,49 @@ def buildHomepage() {
   return homepageTasks
 }
 
+def deployWebUI() {
+  node("elektrad") {
+    docker.withRegistry("https://${REGISTRY}",
+                        'docker-hub-elektra-jenkins') {
+      def backendName = "elektrad"
```

Comment:
keep in mind those names are deployed without elektra context. if i am a system administrator and i see a webd container i no nothing about it.

Reply:
but only on our local registry, right? then the elektra context is there, because it is the elektra docker registry.